content/source/docs/enterprise/api/oauth-clients.html.md

```diff
@@ -170,6 +172,7 @@ Key path                             | Type   | Default | Description
 -------------------------------------|--------|---------|------------
 `data.type`                          | string |         | Must be `"oauth-clients"`.
 `data.attributes.service-provider`   | string |         | The VCS provider being connected with. Valid options are `"github"`, `"github_enterprise"`, `"bitbucket_hosted"`, `"gitlab_hosted"`, `"gitlab_community_edition"`, or `"gitlab_enterprise_edition"`.
+`data.attributes.name`               | string | `null`  | An optional display name for the OAuth Client. If left `null`, the Terraform Enterprise UI will default to displaying the value of `service-provider-display-name` as the OAuth Client's name.
```

Comment:
Is `service-provider-display-name` defined anywhere in our docs? It might be more straightforward to say "default to the display name of the VCS provider".

Reply:
Yeah, it's listed as an attribute in OAuth Client responses

Reply:
I'm not sure that anyone reads the whole JSON in these responses (I certainly never have :)). I think it's more straightforward to describe it the way I did, but up to you.